hack/ci-e2e-debug-fail.sh

```diff
@@ -57,7 +57,7 @@ debug_common() {
     for worker in $(kubectl get node -o name -l node.kubernetes.io/worker 2>/dev/null); do
         echo "::group::journalctl -t kata ($worker)"
         kubectl debug --image quay.io/prometheus/busybox -q -i \
-            "$worker" -- chroot /host journalctl -x -t kata --no-pager
+            "$worker" -- chroot /host journalctl -x -t kata --profile=general --no-pager
```

Comment:
I have never heard of `--profile` option to `journalctl` and my fedora doesn't support it. I tried a kind `kindest/node:v1.30.0` and it doesn't support it either.